Sources/Testing/Traits/TimeLimitTrait.swift

```diff
@@ -54,35 +52,34 @@ public struct TimeLimitTrait: TestTrait, SuiteTrait {
 
 @available(_clockAPI, *)
 extension Trait where Self == TimeLimitTrait {
-  /// Construct a time limit trait that causes a test to time out if it runs for
-  /// too long.
+  /// Constructs a time limit trait that causes a test to time out if it runs
+  /// for too long.
   ///
   /// - Parameters:
   ///   - timeLimit: The maximum amount of time the test may run for.
   ///
   /// - Returns: An instance of ``TimeLimitTrait``.
   ///
   /// Test timeouts do not support high-precision, arbitrarily short durations
-  /// due to variability in testing environments. The time limit must be at
-  /// least one minute, and can only be expressed in increments of one minute.
+  /// due to variability in testing environments. You express the duration in
+  /// minutes, with a minimum duration of one minute.
   ///
-  /// When this trait is associated with a test, that test must complete within
-  /// a time limit of, at most, `timeLimit`. If the test runs longer, an issue
-  /// of kind ``Issue/Kind/timeLimitExceeded(timeLimitComponents:)`` is
-  /// recorded. This timeout is treated as a test failure.
+  /// When you associate this trait with a test, that test must complete within
+  /// a time limit of, at most, `timeLimit`. If the test runs longer, the
+  /// testing library records a
```

Comment:
Nitpick: whitespace seems wonky here?

Reply:
It's because the link ``Issue/Kind/timeLimitExceeded(timeLimitComponents:)`` is really long and forces a wrap. I've got the line-limit marker set to 80 columns, and configured Xcode to use a fixed-width font for documentation markup, and I'm not sure I can do better than this layout.

Reply:
FWIW, we usually just let the long links overflow. The point of wrapping is to give us a sense of how long the documentation will be when rendered, and the link will be rendered shorter anyway.